lib/encoder_impl.cc

```diff
@@ -510,11 +517,10 @@ int encoder_impl::work (int noutput_items,
 	return noutput_items;
 }
 
-encoder::sptr
-encoder::make (unsigned char pty_locale, int pty, bool ms, std::string ps,
-               double af1, double af2, bool tp, bool ta, int pi_country_code,
-               int pi_coverage_area, int pi_reference_number,
-               std::string radiotext) {
+encoder::sptr encoder::make (unsigned char pty_locale, int pty, bool ms,
+                             std::string ps, double af1, double af2, bool tp,
+                             bool ta, int pi_country_code, int pi_coverage_area,
+                             int pi_reference_number, std::string radiotext) {
```

Comment:
OK,  but generally I would put whitespace stuff in separate commits. AFAIS, the formatting of that function differs from the others either way :-/